icon/iiss/prepmanager.go

```diff
@@ -34,28 +34,30 @@ const (
 type PRep struct {
 	*icstate.PRepBase
 	*icstate.PRepStatus
+	*icstate.State
```

Comment:
Remove icstate.State from PRep and inject bonded requirement from outside.

Reply:
I changed bond requirement value injected to prepStatus through prepManager